src/components/v5/frame/PageLayout/types.ts

```diff
@@ -4,7 +4,9 @@ import { type PageHeaderProps } from './partials/PageHeader/types.ts';
 
 export interface PageLayoutProps {
   sidebar: ReactNode;
-  headerProps: PageHeaderProps;
+  headerProps?: PageHeaderProps;
   topContent?: ReactNode;
   header?: ReactNode;
+  // When set to true, the PageLayout will apply sm vs desktop styles. As opposed to md vs desktop styles.
+  restrictLayoutToMobileAndDesktopBreakpoints?: boolean;
```

Comment:
Way better than before @rumzledz 🥇 though I believe it's more an enablement than a restriction 🙈

Reply:
hahaha all right all right let's meet halfway, I shall call it enable! 🙌

Reply:
@mmioana pushed! 🚀  I hope the variable name is much better now 🤞